.github/scripts/ci-test-gc-core.sh

```diff
@@ -0,0 +1,31 @@
+set -e
+
+# run tests that are more specific to the GC
+# this list was built intuitively based on tests that 
+# call the gc manually, have finalizers, have multiple threads, etc.
+
+. $(dirname "$0")/common.sh
```

Comment:
Are you sure it picked up the latest changes? I've added this line which should fix that error. Also note that the log points at line 25, which is now at line 27 after this change.

Reply:
Probably I was running on an old commit. I will rerun it.

Reply:
A rerun from mmtk-core passed: https://github.com/mmtk/mmtk-core/actions/runs/6079604369/job/16494059161?pr=941